oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/instance/InstanceTraffic.java

```diff
@@ -175,6 +175,8 @@ public Metrics toDay() {
     }
 
     public static class PropertyUtil {
+        public static final String NAMESPACE = "namespace";
+        public static final String POD = "pod";
```

Comment:
Could you add comments for these two to show these are key properties for on-demand logs? And `@since`?

Reply:
Added